app/scripts/dimApp.config.js

```diff
@@ -18,6 +18,9 @@
       active: null,
       debug: false
     })
+    .value('dimGlobalFlags', {
+      isExtension: window.chrome && window.chrome.extension
+    })
```

Comment:
I don't think there's any reason to have `dimGlobalFlags` separate from `dimFeatureFlags` is there?